src/OCSError.js

```diff
@@ -2,14 +2,16 @@
  * @class OCSError
  * @classdesc A error class for OCS errors
  * @param   {string}  message   message to be added to the OCSError
+ * @param   {number}  statusCode  statusCode to be added to the OCSError
  * @param   {object}  response  http response
  * @param   {object}  ocs       ocs additional data
 
  */
 
 class OCSError extends Error {
-  constructor (message, response, ocs = null) {
+  constructor (message, statusCode, response, ocs = null) {
```

Comment:
Is this class public/exported? Then this is a breaking change and needs a major version bump 🙈

Reply:
No